deploy/kiali/kiali_cr.yaml

```diff
@@ -331,6 +336,8 @@ spec:
 # Note that only 'metadata.annotations' and 'spec' is valid and only they will
 # be used to override those same sections in the created resource. You can define
 # either one or both.
+# Note that `metadata.labels` is not allowed - you cannot override the labels; to add
+# labels to the default set of labels, use the "additional_labels" setting.
```

Comment:
Why can't you set `metadata.labels` if you want to override the defaults completely?

Reply:
We do not want users to override the defaults completely (which is why we have it separate like this). We still want our own labels still on the resource (things like "app: kiali") - we want these labels so they identify the resources Kiali creates. All our created resources have the same labels on them.